detections/application/splunk_risky_command_abuse_ml_detection.yml

```diff
@@ -0,0 +1,67 @@
+name: Splunk Risky Command Abuse ML Detection
+id: 19d0146c-2eae-4e53-8d39-1198a78fa9ca
+version: 1
+date: '2022-05-27'
+author: Abhinav Mishra, Kumar Sharad and Xiao Lin, Splunk
+type: Anomaly
+datamodel:
+- Splunk_Audit
+description: 'This detection utilize machine learning model named "risky_command_abuse" trained from "Splunk 
+  Risky Command Abuse ML Model Baseline". It should be scheduled to run every hour 
+  to detect whether a user has run searches containing risky keywords with abnormal long running time in the 
+  past two hours, comparing with his/her past seven days history. '
+search: '| tstats values(Search_Activity.user) AS user, sum(Search_Activity.total_run_time) AS run_time,
+  values(Search_Activity.search) as searches, count 
+  FROM datamodel=Splunk_Audit.Search_Activity WHERE (Search_Activity.user!="") 
+  AND (Search_Activity.total_run_time>1) AND (earliest=-2h@h latest=now) 
+  AND (Search_Activity.search IN ("*| runshellscript *", "*| collect *","*| delete *", "*| fit *", "*| outputcsv *", 
+  "*| outputlookup *", "*| run *", "*| script *", "*| sendalert *", "*| sendemail *", "*| tscollect *")) 
+  AND (Search_Activity.search_type=adhoc) AND (Search_Activity.user!=splunk-system-user) 
+  BY _time, Search_Activity.user span=1h 
+  | apply risky_command_abuse
+  | fields _time, user, searches, run_time, IsOutlier(run_time)
+  | rename IsOutlier(run_time) as isOutlier, _time as timestamp
+  | where isOutlier>0.5 | `splunk_risky_command_abuse_ml_detection_filter`'
+how_to_implement: This detection depends on MLTK and assumes Splunk audit data model is available. 
+known_false_positives: Benigh search of long run time will be flagged as positive (false), if the run time of 
+  a search exceeds the boundaries of outlier defined by the fitted density function model. 
+references:
+- https://docs.splunk.com/Documentation/Splunk/latest/Security/SPLsafeguards#Commands_that_trigger_the_warning
+tags:
+  analytic_story:
+  - Splunk Vulnerabilities
+  asset_type: Web Server
+  cis20:
+  - CIS 3
+  - CIS 6
+  confidence: 40
+  context:
+  - Source:Endpoint
+  dataset:
+  - https://github.com/splunk/attack_data/raw/master/datasets/attack_techniques/T1203/search_activity.txt
+  impact: 50
+  kill_chain_phases:
+  - Actions on Objectives
+  message: $user used abnormally long run time for searches with risky command.
```

Comment:
rename the message to:
"Abnormally long run time for risk SPL command seen by user $user$"

Reply:
done